event_reader/src/event_reader.py

```diff
@@ -7,6 +7,7 @@
 import psycopg2 as pg
 from sqlalchemy import create_engine
 
+# touch to add comments
```

Comment:
Storing the original payload is nice for historical archive purposes but I don't think it's the best approach for a real time alerting system. There are a few issues:

* Query evaluation performance is one of the most important parts of a real time alerting system. I'd have to do some comparison benchmarks, but I would be working from the assumption that index size & query performance would be worse on unnormalized JSONB data than normalized columns
* Last time I looked into this JSONB indexes aren't as efficient for the query planner because they don't offer the same level of static collection
* The data is not normalized so you need a new query per payload
* If the payload ever changes you will need multiple queries per pair and some way to switch on that e.g. a version number